docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,255 @@
+# Instrumenting Android Crashes with Glean
+
+One of the things that we might want to collect some data on in the applications we create is crashes.  Every application will
+have them from time to time.  Sometimes they are the application’s fault, sometimes it’s the fault of the OS, but they happen
+and we usually want to know more about them so we can fix them (if possible).  So let’s take a look at how we can use Glean to
+instrument an Android application and record some data when it crashes.
+
+## Before We Start
+
+There are a few things that we need to have installed in order to get ready.  The biggest one of those is [Android Studio](https://developer.android.com/studio/),
+which (if you include the Android SDK) can take a little while to download and get installed.  Second, I’m going to recommend
+having the [Book of Glean](https://mozilla.github.io/glean/book/) open and available as we will be referring to it throughout
```

Comment:
We refer to it as "The Glean SDK Book" in the readme. We did discuss the naming a while ago but I forgot what the decision was.

Reply:
also given that this chapter is _inside_ that book we might want to rephrase this?